rollup.config.js

```diff
@@ -8,7 +8,6 @@ const json = require('rollup-plugin-json')
 const babel = require('rollup-plugin-babel')
 const replace = require('rollup-plugin-replace')
 const { uglify } = require('rollup-plugin-uglify')
-const filesize = require('rollup-plugin-filesize')
```

Comment:
This had a dep "brotli"-something that was throwing an error because of engine checks for a different node version